### PR TITLE
poetry 1.5.0

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by chenrui333** [comment]
```
  ==> /opt/homebrew/Cellar/poetry/1.5.0/bin/poetry add requests
  
  Command ['python', '-I', '-W', 'ignore', '-'] errored with the following return code 2
  
  Error output:
  Unknown option: -I
```

File: Formula/poetry.rb [hunk 1]
```diff
@@ -247,6 +242,8 @@ def install
     # https://github.com/Homebrew/homebrew-core/pull/109777#issuecomment-1248353918
     ENV["PYTHON_KEYRING_BACKEND"] = "keyring.backends.null.Keyring"
 
+    ENV.prepend_path "PATH", Formula["python@3.11"].opt_libexec/"bin"
```
Comment (carlocab) [inline Formula/poetry.rb:245]: Do users have to do this too to be able to use Poetry? If so we either need to add `caveats` or adjust the formula so they don't need to do this.

**Review by carlocab — CHANGES_REQUESTED** [review]

File: Formula/poetry.rb (hunk 1 above)
Comment (chenrui333) [reply Formula/poetry.rb:245]: yeah, this was indeed a bit surprising to me as well.

**Comment by carlocab** [comment]
Thanks for the update. I think we'll wait for 1.5.1 then.

**Comment by chenrui333** [comment]
I can also pull that patch and see if works for the regression test.

**Review by fxcoudert — APPROVED** [review]

**Review by carlocab — APPROVED** [review]

**Pull request merged**